package.json

```diff
@@ -76,6 +76,7 @@
     "markdown-to-jsx": "^7.3.2",
     "micro-observables": "^1.7.2",
     "notebook": "file:vendor/notebook",
+    "re-resizable": "^6.9.11",
```

Comment:
Previously, `react-virtuoso` and [TanStack Virtual](https://tanstack.com/virtual/v3) were [suggested by Emily](https://hpe-aiatscale.slack.com/archives/CKQ3EV0G6/p1687295973804119). Have we decided go with `react-window` with `re-resizable`?